test/ufuzz.js

```diff
@@ -575,7 +575,7 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
             createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
       case 28:
         return '(' + createObjectLiteral(recurmax, COMMA_OK, stmtDepth, canThrow) + '[' +
-            createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
+            KEYS[rng(KEYS.length)] + "]) ";
```

Comment:
AFAICT the very nature of `--mangle-props` means that if we ever give it computed (i.e. non-constant) property values, it screws up.

I can make this conditional on whether we are employing `mangleProperties` or not, to enable the safer options in `ufuzz.json` to remain capable of being fuzzed with more randomness. Would that be good enough?